corehq/apps/cloudcare/static/cloudcare/js/formplayer/router.js

```diff
@@ -209,6 +209,9 @@ hqDefine("cloudcare/js/formplayer/router", function () {
             // force manual search in split screen case search for workflow compatibility
             sidebarEnabled ? { forceManualSearch: true } : {}
         );
+
+        var queryObject = _.extend(queryObject, initiator === "dynamicSearch" ? { initiatedBy: "dynamic_search"} : {});
```

Comment:
Why the change from camel case to snake case? Couldn't this be:
```suggestion
        var queryObject = _.extend(queryObject, {initiatedBy: initiator});
```

Reply:
Changed